### PR TITLE
DL-6451: Remove play-frontend-govuk dependency

### PR DESCRIPTION
**Pull request by nadinJ**
DL-6451: Remove play-frontend-govuk dependency

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -44,8 +44,8 @@ lazy val microservice: Project = Project(appName, file("."))
     ),
     TwirlKeys.templateImports ++= Seq(
       "uk.gov.hmrc.govukfrontend.views.html.components._",
-      "uk.gov.hmrc.govukfrontend.views.html.helpers._",
-      "uk.gov.hmrc.hmrcfrontend.views.html.{components => hmrcComponents}",
+      "uk.gov.hmrc.hmrcfrontend.views.html.components",
```
Comment (paulanderson1234) [inline build.sbt]: should this be components._ ?

File: app/views/layout.scala.html [hunk 2]
```diff
@@ -42,6 +42,12 @@
 
 @(pageTitle: Option[String] = None, scripts: Option[Html],service: String)(contentBlock: Html)(implicit request: Request[_], messages: Messages, appConfig: ApplicationConfig)
 
+@headBlock = {
```
Comment (paulanderson1234) [inline app/views/layout.scala.html]: Where did this come from ?
Was it something that we believe was missed previously ?

Comment (nadinJ) [reply app/views/layout.scala.html]: I've added it to keep uniformity, the rest of our frontends have it

File: build.sbt (hunk 1 above)
Comment (paulanderson1234) [reply build.sbt]: Resolved as change applied

File: app/views/layout.scala.html (hunk 2 above)
Comment (paulanderson1234) [reply app/views/layout.scala.html]: resolved as this has been removed as not required

**Pull request merged**